lib/Reader/readerir.cpp

```diff
@@ -7462,11 +7462,15 @@ void GenIR::addPHIOperand(PHINode *Phi, Value *NewOperand,
     LLVMBuilder->restoreIP(SavedInsertPoint);
   }
 
-  int BlockIndex = Phi->getBasicBlockIndex(NewBlock);
-  if (BlockIndex >= 0)
-    Phi->setIncomingValue(BlockIndex, NewOperand);
-  else
-    Phi->addIncoming(NewOperand, NewBlock);
+  bool FoundBlockOperand = false;
```

Comment:
> We do have several warnings about other initialized but not referenced variables in release builds. I'll fix those separately.

Thanks.  If that makes us warning-clean, we should probably start building with `-DLLVM_ENABLE_WERROR=ON` in the lab.